### ISSUE TITLE
Update dark mode contrast between background and text

### ISSUE THREAD

**Issue by thatblindgeye** [TheOdinProject/theodinproject#3033]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `brief description of request` format, e.g. `Add dark mode to website`

The following checkbox is OPTIONAL:

-   [ ] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

There have been some complaints regarding the contrast between the background and text in dark mode. Currently it looks like the body background color uses `rgb(18,18,18)` while primary text uses peak white at `rgb(255,255,255)` which might be what's causing the issue.

A couple of suggestions have been to look at Twitter and Reddit for their dark theme since they are primarily reading based much like TOP. Another option is to use dark reader's values as those who have complained about TOP's dark mode said they use dark reader without much issue.

In dark reader the body color is `rgb(24, 26, 27)` while primary text color looks to be `rgb(214, 211, 205)` (in general I think it's advised to not use peak black for backgrounds or peak white for text due to that high contrast for a simple dark theme).

**2. Acceptance Criteria:**

- [ ] Update the contrast between background(s) and text color

**3. Additional Information:**

Discussion on Discord starts about here and goes down several messages: https://discord.com/channels/505093832157691914/505093832157691916/992078735153037332

**Comment by joayo13** [comment]
Hi, I'd like to help with this issue if possible. My idea was to use the same background-color/ text color values that reddit uses in threads. It isn't too drastic a change from what the website already uses so I thought it was a good fit.

**Comment by KevinMulhern** [comment]
If it's just updating the color values that should be fine. But if it involved changing markup or classes in any of the views it probably isn't worth the effort. We're working on converting everything over to Tailwind and that will involve a complete overhaul of the dark mode system https://github.com/TheOdinProject/top-meta/projects/7#card-76439405
